src/OnboardingSPA/pages/Steps/BasicInfo/Sidebar/LearnMore/contents.js

```diff
@@ -30,23 +30,13 @@ const getContents = ( brandName ) => {
 					description: sprintf(
 						/* translators: 1: Brand 2: Site */
 						__(
-							`Enter the best way for WordPress, web browsers and search engines to identify your %s %s by entering your Title, Description and Logo.`,
+							`Loading your site details, logo and social graph helps not just launch your site but have it found looking great in feeds.`,
```

Comment:
@0aveRyan should all occurrences of `site` be replaced with `store` dynamically here? We use `store` for everything else in basic info.
![Screenshot 2022-12-12 at 12 24 41 PM](https://user-images.githubusercontent.com/38878906/206980208-1209f093-ea01-405f-9129-64bcc257b7c3.png)

Reply:
Yes, fine if we fix post-release though let's just note.